cola/__main__.py

```diff
@@ -16,5 +18,10 @@ def run():
     main.main()
 
 
+def run_dag():
+    args = ["dag"] + sys.argv[1:]
+    main.main(args)
```

Comment:
I think it'd be better to `from cola import dag` and then `dag.main` here, which matches what `bin/git-dag` does.